index.html

```diff
@@ -12,7 +12,6 @@
   </head>
   <body>
     <section class="container bg-light p-0 m-0">
-      <!-- top panel -->
       <header class="m-0 p-0 container">
```

Comment:
- [ ] Good job on the design but for this project you are supposed to stick to the [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template as much as possible.
  - Please place more gap between your menu items and make sure it matches the design on the template.
  - Please remove the background color from your campaign button and place the border as on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/158639927-0b6ba422-ec44-467c-a5ea-6e69fb5bb9c4.png)

## Behnace design

![image](https://user-images.githubusercontent.com/83803414/158640024-7049dd79-1fc9-4382-bd02-22d26e84f3cb.png)